components/admin_console/admin_definition.jsx

```diff
@@ -5148,7 +5162,7 @@ const AdminDefinition = {
                         help_text_default: 'When true, Mattermost allows compliance reporting from the **Compliance and Auditing** tab. See [documentation](!https://docs.mattermost.com/administration/compliance.html) to learn more.',
                         help_text_markdown: true,
                         isHidden: it.not(it.licensedForFeature('Compliance')),
```

Comment:
Do we need the read permission here?

Reply:
@BenCookie95 No because if they don't have read permission, the entire section will not show up. We won't need to hide individual options. We are only required to disable it if they don't have write permission.